src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_layout.html.twig

```diff
@@ -800,7 +800,7 @@
 
 {% block form_help %}
   {% if help %}
-    <small class="form-text">{{ help }}</small>
+    <small class="form-text">{{ help|nl2br }}</small>
```

Comment:
Be aware that this should have no effect on your PR since you're not using this file in the forms you edited (this file is the old one which will be deleted once we switch to the new form theme).

The `form_help` block is declared here: https://github.com/PrestaShop/PrestaShop/blob/6ca37a71cc51668f45732d3616f2e6ab71875ff2/src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig#L821-L828

I know this may be confusing, but the `bootstrap_4_layout.html` file referenced in `prestashop_ui_kit_base.html.twig` actually points to `vendor/symfony/symfony/src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig`, not to this file.